shell/platform/darwin/ios/framework/Source/accessibility_bridge.h

```diff
@@ -91,7 +91,7 @@ class AccessibilityBridge final : public AccessibilityBridgeIos {
 
   FlutterViewController* view_controller_;
   PlatformViewIOS* platform_view_;
-  const std::shared_ptr<PlatformViewsController> platform_views_controller_;
+  __weak FlutterPlatformViewsController* platform_views_controller_;
```

Comment:
is it intended to change to weak?

Reply:
Yes. The owner of the `PlatformViewsController` is `FlutterEngine` and the lifetime of `PlatformViewsController` is tied to that. Conveniently, `AccessibilityBridge` is also owned by the engine and its lifetime is tied to that, so this was always unnecessary.